src/rpc/rawtransaction.cpp

```diff
@@ -933,6 +935,10 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
             result_0.pushKV("reject-reason", state.GetRejectReason());
         }
     }
+    // Push tx fee, only if test_accept_res == true
+    else {
```

Comment:
double negatives are hard to follow (probably explains why you had to add a dev comment). Easier would be to write

```cpp
if (test_accept_res) {
 push fee
} else {
 puls reject reason
}
```

for self-documenting code

Reply:
makes sense, done.